plugins/automap/README.md

```diff
@@ -0,0 +1,9 @@
+# @flatfile/plugin-automap
+
+The purpose of this plugin is to provide automapping for headless workflows. 
+
+`npm i @flatfile/plugin-automap`
+
+## Get Started
+
+Follow [this guide](https://flatfile.com/docs/plugins/transform/automap) to learn how to use the plugin.
```

Comment:
❔ Will this be the right link?

Reply:
actually, does transform feel wrong here? We could make a new section for automations: https://flatfile.com/docs/plugins/automation/automap